src/services/services.js

```diff
@@ -0,0 +1,15 @@
+import fetch from '../utils/fetch.js';
+
+const SERVICE_ID_VALIDATION_REGEX = /^[a-zA-Z0-9\s\-'.!\?]+$/; // eslint-disable-line no-useless-escape
```

Comment:
According to [the documentation](https://docs.opentermsarchive.org/contributing-terms/#service-id), spaces and all punctuation except `:` `/` `\` are supported. For example, here, `;` would not pass the validation regex while it is a valid service ID.